docs/document/content/user-manual/shardingsphere-proxy/cdc/build.cn.md

```diff
@@ -0,0 +1,160 @@
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| MySQL     | [mysql-connector-java-8.0.11.jar]( https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.49/mysql-connector-java-5.1.49.jar ) | [Connector/J Versions]( https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.11/mysql-connector-java-8.0.11.jar ) |
+| openGauss | [opengauss-jdbc-3.1.1-og.jar]( https://repo1.maven.org/maven2/org/opengauss/opengauss-jdbc/3.1.1-og/opengauss-jdbc-3.1.1-og.jar )        |                                                                                                  |
+
+4. 启动 ShardingSphere-Proxy：
+
+```
+sh bin/start.sh
+```
+
+5. 查看 proxy 日志 `logs/stdout.log`，看到日志中出现：
+
+```
+[INFO ] [main] o.a.s.p.frontend.ShardingSphereProxy - ShardingSphere-Proxy Cluster mode started successfully
+```
+
+确认启动成功。
+
+## CDC Client 使用手册
+
+用户可以通过 CDC Client 和服务端进行交互，CDC Client 的依赖很轻，只包含 netty 以及 CDC 协议相关的依赖。
+
+有两种方式可以引入 CDC Client
+
+1. 源码编译，CDC Client 在编译 Proxy 的时候会一同编译，在 kernel/data-pipeline/scenario/cdc/client/target 目录下可以找到编译后的 jar 文件
+2. 从 maven 仓库获取，[Shardingsphere Data Pipeline CDC Client](https://mvnrepository.com/artifact/io.github.greycode/shardingsphere-data-pipeline-cdc-client)
+
+### CDC Client 介绍
+
+`org.apache.shardingsphere.data.pipeline.cdc.client.CDCClient` 是 CDC Client 的入口类，用户可以通过该类和 CDC Server 进行交互。主要的和新方法如下。
+
+| 方法名                                                | 返回值 | 说明 |
+|----------------------------------------------------| --- | --- |
+| await()                                            | void | 阻塞 CDC 线程，await channel 关闭 |
+| close()                                            | void | 关闭 channel |
+| connect()                                          | void | 和服务端进行连接 |
+| login (CDCLoginParameter parameter)                | void | 登陆验证 |
+| startStreaming (StartStreamingParameter parameter) | java.lang.String （CDC 任务唯一标识）        | 开启 CDC 订阅 |
+| restartStreaming (java.lang.String streamingId)    | void | 重启订阅 |
+| stopStreaming (java.lang.String streamingId)       | void | 停止订阅 |
+
+
+### CDC Client 使用示例
+
+目前 CDC Client 只提供了 Java API，用户需要自行实现数据的消费逻辑。
+
+1. 引入 CDC Client
+
+```xml
+<dependency>
+    <groupId>io.github.greycode</groupId>
+    <artifactId>shardingsphere-data-pipeline-cdc-client</artifactId>
+    <version>${version}</version>
+</dependency>
+```
+
+2. 启动 CDC Client
+
+参考 [Example](https://github.com/apache/shardingsphere/blob/master/kernel/data-pipeline/scenario/cdc/client/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/client/example/Bootstrap.java) 启动 CDC Client。
```

Comment:
Could we move CDC Client steps to usage document? Since if proxy server in usage document does not start, users could not start CDC Client.